<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Game of life</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Timer Started</string>
    </property>
   </widget>
   <widget class="QLabel" name="counterLabel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Time: 0</string>
    </property>
   </widget>
   <widget class="QSlider" name="timerStepSlider">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>0</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>30</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="value">
     <number>250</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
    <property name="tickPosition">
     <enum>QSlider::TicksAbove</enum>
    </property>
    <property name="tickInterval">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Time step:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="resetButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QPushButton" name="randomButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>10</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Random</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fillButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Fill</string>
    </property>
   </widget>
   <widget class="QWidget" name="glWidget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>391</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
